setup.cfg

```diff
@@ -12,5 +12,5 @@ exclude =
     __pycache__
     .git
     htmlcov
-max-line-length = 120
+max-line-length = 92
```

Comment:
I'm curious what this max-line-length applies to (this repo, all repos, JSONs formed from code, etc?) and what the motivation was for this change

Reply:
Most of these tools search up the directory hierarchy, so this would only affect this repo, and it specifically applies to flake8's warning about line length.

I was setting that so that flake8 is more consistent with black.